tests/io/hudi/test_table_read.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import annotations
+
+import pyarrow as pa
+import pytest
+
+import daft
+
+PYARROW_LE_10_0_0 = tuple(int(s) for s in pa.__version__.split(".") if s.isnumeric()) < (10, 0, 0)
+pytestmark = pytest.mark.skipif(PYARROW_LE_10_0_0, reason="hudi only supported if pyarrow >= 10.0.0")
+
+
+def test_hudi_read_table(unzip_table_0_x_cow_partitioned):
```

Comment:
Is there a larger suite of tests that we should be using here to understand our compatibility with various Hudi corner-cases?

Reply:
i didn't working on more test cases but i agree there should be. How about adding those iteratively?

Reply:
yes adding more iteratively is completely fine!

Reply:
cool. I'm tracking these internally. if you are also tracking this and the follow-up work in any issue, feel free assign to me.

Reply:
Great! Could you create an [issue](https://github.com/Eventual-Inc/Daft/issues) titled "Hudi integration todo list"? Would love to track it there. We can enumerate any tasks as a sub-task of the issue (Using the `- [ ]` syntax in the issue)